build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -322,6 +323,107 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
+        
+        [Target]
+        [BuildPlatforms(BuildPlatform.Windows)]
+        public static BuildTargetResult CompileUwpHosts(BuildTargetContext c)
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                return c.Failed();
+
+            var configuration = c.BuildContext.Get<string>("Configuration");
+            string platform = c.BuildContext.Get<string>("Platform");
+
+            // cmake generated build file location artifacts\<config>\uwp\cmake
+            var cmakeOut = Path.Combine(Dirs.Uwp, "cmake");
+            Rmdir(cmakeOut);
+            Mkdirp(cmakeOut);
+
+            // cmake generated build file location artifacts\<config>\uwp
+            var cmakeInstallPrefixDir = Dirs.Uwp;
+
+            // Run the build
```

Comment:
This comment is inconsistent with what the next line is actually doing, and a comment probably isn't needed because it's fairly obvious what the next line is doing.